datasets/svhn/README.md

```diff
@@ -18,7 +18,7 @@ task_categories:
 - image-classification
 - object-detection
 task_ids:
-- single-label-image-classification
+- object-detection-other-digit-detection
```

Comment:
Would remove this and define task ids as `task_ids: []`

Reply:
done !